commons-app/apps-android-commons
# Fixes #5531 Circular Progress bar keeps on moving when nomination for deletion is successful.

## Fixes
* Fixes #5531. 

## Description
This pull request addresses an issue where the circular progress bar remains active and keeps moving even after a nomination for deletion is successful.  The fix ensures that the progress bar is hidden and the UI is updated correctly upon successful deletion also the Snackbar is shown when the nomination is done successfully.

## Changes
- Modified the `onDeleteClicked()` & `onDeleteClickeddialogtext`  & created a `handleDeletionResult()` for handling what to do on success, created a `handleDeletionError()` for handling what to do on failure. 
- Ensured that UI updates are performed on the UI thread to prevent potential `CalledFromWrongThreadException`.

## Test Plan
### Steps to Test
1. Open the app.
2. Open any contribution.
3. Click on the nomination for deletion.
4. Observe the progress bar behavior after the nomination process completes.

### Expected Results
- The circular progress bar should disappear, indicating that the process has completed.
- The user should see a confirmation message indicating the successful nomination for deletion.

## Screenshots


## Additional Information
- Tested on both emulators and physical devices to verify the fix.
